dist/mmcif_pdbx_v50.dic

```diff
@@ -143055,7 +143082,9 @@ save__reflns_shell.gphl_percent_possible_ellipsoidal_all
 
 save__reflns_shell.gphl_percent_possible_obs_anomalous
     _item_description.description
-;              GPhL TBD
+;              Completeness of anomalous data within the diffraction
```

Comment:
Possibly "Percentage  of all possible Friedel pairswithin this shell (as defined by its diffraction limits) that are observed" ? Or is it "that are meaasured"?

Reply:
Given the item name (_obs) it should be "observed", yes.

However, we are computing the same statistic for 

1. all measurements to the full diffraction limit defined by the experiment (detector etc),
2. all measurements to the diffraction limit of the data after an "observation criterion" has been applied (e.g. via the STARANISO analysis) - which is not using only observations
3. all observations

The item naming has been inherited from 
- [_reflns.percent_possible_obs](https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_reflns.percent_possible_obs.html) (used in 85%)
- [_reflns_shell.percent_possible_obs](https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_reflns_shell.percent_possible_obs.html) (used in <1%)
- [_reflns_shell.percent_possible_all](https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_reflns_shell.percent_possible_all.html) (used in 71%)

In principle we have a distinction between "observed reflections" (metrics with "_obs" in item name) or "measurements" (metrics with "_all" in item name) - but that is not based on [_reflns.observed_criterion](https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_reflns.observed_criterion.html), which is only used in 0.001% of PDB entries anyway. So we could have the following situations:

-  we have measurements for which a subset is defined as being "observed" (through whatever criterion/method): we can compute the "_obs" and the "_all" variant and they will be different.
- we have measurements that are clearly defined to be all observed: can really only compute the "_obs" variant
- we have measurements and no notion about "observed": could compute both variants (and they will be identical)?